### PR TITLE
Flake8 fix

### PR DESCRIPTION
**Pull request by nwu63**
## Purpose
This PR fixes the flake8 violations as a result of the updated configurations. I decided it was probably not worth fixing some of the files, so I added them to the flake8 ignore list.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

## Testing
N/A

## Checklist
_Put an `x` in the boxes that apply._

- [x] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [x] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: tests/reg_tests/test_adjoint.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def twist(val, geo):
     def span(val, geo):
         # Span
         C = geo.extractCoef("wing")
-        for i in range(len(C) - 1):
+        for _i in range(len(C) - 1):
```
Comment (nwu63) [inline tests/reg_tests/test_adjoint.py:57]: I have no idea whether the original code was correct since I've not used the span function before. Why do we not use the loop variable here?

File: adflow/pyADflow.py [hunk 2]
```diff
@@ -5254,25 +5254,23 @@ def _getSpecialOptionLists(self):
         # pythonOptions do not get set in the Fortran code.
         # They are used strictly in Python.
 
-        pythonOptions = set(
```
Comment (marcomangano) [inline adflow/pyADflow.py:5257]: Why did you change this from a set to a "flat" dictionary?

File: tests/reg_tests/test_adjoint.py (hunk 1 above)
Comment (marcomangano) [reply tests/reg_tests/test_adjoint.py:57]: This looks like a bug to me, we should move all the `refAxis` points, the loop is clearly useless otherwise

**Review by marcomangano — COMMENTED** [review]
How much effort would it be to fix the other files you added to `extend-exclude`? I agree they look low priority, but we might want to fix them once for all at some point?

File: adflow/pyADflow.py (hunk 2 above)
Comment (nwu63) [reply adflow/pyADflow.py:5257]: They are the equivalent syntax: `{'a', 'b'} == set(('a', 'b'))`.

File: tests/reg_tests/test_adjoint.py (hunk 1 above)
Comment (nwu63) [reply tests/reg_tests/test_adjoint.py:57]: Yeah that was my thought also, but I don't really know how to fix it. In any case this highlights how checking unused loop variables can prevent bugs.

Comment (marcomangano) [reply tests/reg_tests/test_adjoint.py:57]: @joanibal it looks like you were involved in #47 when we updated the testing section, do you recall how is this DV supposed to work?

**Comment by nwu63** [comment]
> How much effort would it be to fix the other files you added to `extend-exclude`? I agree they look low priority, but we might want to fix them once for all at some point?

They are not too bad, but other than `fort_depend` and `autoEdit` the others are probably going to be deprecated soon (if not already) so I didn't feel like fixing them.

**Comment by marcomangano** [comment]
> They are not too bad, but other than `fort_depend` and `autoEdit` the others are probably going to be deprecated soon (if not already) so I didn't feel like fixing them.

Ok I see, let's just don't forget about them so when they'll be removed we can also clean up the few remaining files

**Review by marcomangano — APPROVED** [review]

**Comment by marcomangano** [comment]
@joanibal @ArshSaja @sseraj could any of you check this and merge if there are no issues?
Also please take a look at the potential "bug" in `test_adjoint` I commmented above

File: tests/reg_tests/test_adjoint.py (hunk 1 above)
Comment (sseraj) [reply tests/reg_tests/test_adjoint.py:57]: I agree that this is a bug. The span variable should be something like:
```
        def span(val, geo):

            C = geo.extractCoef("wing")
            s = geo.extractS("wing")

            for i in range(len(C)):
                C[i, 2] = C[i, 2] + s[i] * val[0]

            geo.restoreCoef(C, "wing")
```

However, I don't think this is affecting the test coverage and fixing it would require retraining the test. I can create an issue and then merge this in as is.

Comment (marcomangano) [reply tests/reg_tests/test_adjoint.py:57]: Agreed @sseraj

Comment (sseraj) [reply tests/reg_tests/test_adjoint.py:57]: The issue is at #163

**Review by sseraj — APPROVED** [review]

File: tests/reg_tests/test_adjoint.py (hunk 1 above)
Comment (joanibal) [reply tests/reg_tests/test_adjoint.py:57]: I have no explanation for this. It was probably a temporary change for debugging that accidentally became permanent.
